debug/dprof-dump.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 const fs = require('fs')
 const analysis = require('../analysis/index.js')
-const getLoggingPaths = require('../collect/get-logging-paths.js')
+const getLoggingPaths = require('@nearform/clinic-common/lib/get-logging-paths')('bubbleprof')
```

Comment:
This should export this function directly, or put it in the top-level.

Reply:
You mean I should export this from e.g. the index.js file instead of requiring it in different files?

Reply:
`lib` should not be part of the required path. You can either:

a) move the file to top-level in clinic-common
b) export it from `index.js`

Reply:
The latter is more stable over time.

Reply:
Addressed.